data/2022_23/seminare/P-MAT/P-MATz1.yml

```
@@ -0,0 +1,33 @@
+# Nazov udalosti
+name: P-MAT 1.séria
+
+# Vedne obory (1+) pre danu akciu. Mozne hodnoty: mat, fyz, inf a other.
+sciences:
+  - mat
+
+# Typ akcie. Moznosti: sutaz, seminar, sustredenie, vikendovka, tabor, olympiada alebo prednasky
+type: seminar
+
+# Informacie k datumom.
+date:
+  # Zaciatok udalosti alebo datum udalosti, ak trva iba den. Format YYYY-MM-DD
+  start: "2022-03-10"
+
+# nazov organizatora alebo zoznam organizatorov
+organizers:
+  - p-mat
+
+# Odkaz na stranku pre viac info (optional)
+link: https://www.p-mat.sk/details/pikomat/
```

neviem či nechcú radšej

```suggestion
link: https://pikomat.sk
```